tools/parse_soundfont.py

```diff
@@ -1096,8 +1101,11 @@ def process_oot_instruments(self):
                     
     def process_percussion_set(self):
         for drum in self.processed_percussions:
-            start = drum.lowrange - 9
-            end = drum.maxrange - 9
+            # Is there a specific reason to subtract X from the range?
+            # The tuning calculation accepts MIDI note values by default, using "Zelda" note values
+            # or any other note value does nothing but add onto the calculation needlessly...
+            start = drum.lowrange #-9
```

Comment:
the - 9 is actually supposed to be -21 that was accidentally committed at 5am, but its there because the games code starts indexing the keys at midi key 21 so if a drums key range was 21 the drum index would be 0, not 21. sequence 64 subtracts 21 from every midi key when generating the sequences because of this so the sound font and sequence wouldn't be aligned.